src/OpenLoco/Vehicles/Vehicle.h

```diff
@@ -245,6 +246,8 @@ namespace OpenLoco::Vehicles
         bool update();
         VehicleStatus getStatus() const;
         OrderRingView getCurrentOrders() const;
+        bool isPlaced() const { return tile_x != -1 && !(var_38 & Flags38::isGhost); }
```

Comment:
Reminder that we should go through and use this in the 100's of places we do this check.